src/services/expenses.service.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+const { Expense } = require('../module/expense.moduls');
+
+function normalize({ id, userId, spentAt, title, amount, category, note }) {
+  return {
+    id,
+    userId,
+    spentAt,
+    title,
+    amount,
+    category,
+    note,
+  };
+}
+
+const getAll = () => {
+  return Expense.findAll({
+    order: ['createdAt'],
+  });
+};
+
+const getById = (id) => {
+  return Expense.findByPk(id);
+};
+
+const postExpenses = ({
+  userId,
+  spentAt,
+  title,
+  amount,
+  category,
+  note,
+}) => {
+  return Expense.create({
+    userId, spentAt, title, amount, category, note,
+  });
+};
+
+const update = ({
+  id,
+  data,
+}) => {
+  return Expense.update(data, {
+    where: { id: id },
```

Comment:
```suggestion
    where: { id },
```
**PLS DO THE SAME FOR ALL SUCH CASES IN THE PROJECT**